.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: false,
+    node: true,
+  },
+  plugins: ['node', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:node/recommended',
+    'plugin:prettier/recommended',
```

Comment:
I think if you use `eslint-config-prettier`, you just specify `"prettier",` here, and it'll correctly configure it for you.